python/testing.rst

```diff
@@ -12,6 +12,11 @@ See :doc:`/coding/unit-test-policy` for an overview of LSST Stack testing.
 LSST tests should be written using the `unittest` framework, with default test discovery, and should support being run using the `pytest`_ test runner as well as from the command line.
 If you want to jump straight to a full example of the standard LSST Python testing boilerplate without reading the background, read the :ref:`section on memory testing <py-test-mem>` later in this document.
 
+.. note::
```

Comment:
This note seems out of place. It'd try to put this note next to the boilerplate example of a test file. I'm not entirely sure what that would be. Either in "Introduction to unittest" or "Memory and file descriptor leak testing" (which is where the full boilerplate seems to be tucked into).

Reply:
I'll have a look. Reminds me that the memory test no longer tests memory...